environments/openshift/manifests/observatorium-template.yaml

```diff
@@ -878,6 +878,218 @@ objects:
         - name: telemeter-server-tls
           secret:
             secretName: telemeter-server-shared
+- apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Due to a known limitation, we cannot create clusterroles for the rbac API, so I'd suggest you use the same one as infogw-proxy is using, for now.